### ISSUE TITLE
Please update instructions for running version 0.6.0

### ISSUE THREAD

**Issue by jbax** [input-output-hk/jormungandr#950]
**Describe the bug**
Node setup instructions are out of date and the sample `node-config.yaml` file in the [How to install Jormungandr Networking (Linux/macOS)](https://iohk.zendesk.com/hc/en-us/articles/360036898153) guide does not work anymore. Currently the guide tells us to create the config file with:

```
log:
  format: "plain"
  level: "info"
  output: "stderr"
p2p:
  listen_address: "/ip4/0.0.0.0/tcp/3100"
  public_address:
  topics_of_interest:
    blocks: "high"
    messages: "high"
  trusted_peers:
    - "/ip4/3.123.177.192/tcp/3000"
    - "/ip4/3.123.155.47/tcp/3000"
    - "/ip4/52.57.157.167/tcp/3000"
    - "/ip4/3.112.185.217/tcp/3000"
    - "/ip4/18.140.134.230/tcp/3000"
    - "/ip4/18.139.40.4/tcp/3000"
    - "/ip4/3.115.57.216/tcp/3000"
rest:
  listen: "127.0.0.1:3101"
storage: "/tmp/jormungandr"
```

This used to work prior to version 0.6.0, but after updating to this version I get the following error:

```
Error in the overall configuration of the node
 |-> Error while parsing the node configuration file: p2p.trusted_peers[0]: invalid type: string "/ip4/3.123.177.192/tcp/3000", expected struct TrustedPeer at line 12 column 7
 |-> p2p.trusted_peers[0]: invalid type: string "/ip4/3.123.177.192/tcp/3000", expected struct TrustedPeer at line 12 column 7

```

Which would be OK if we had a way to know what to change in the `node-config.yaml` file so things can work again. There is not a single note or guide providing this information and it's impossible to start a node without knowing what to do.

Can you guys please update the instructions ASAP?

My jormungandr and jcli versions:
```
jcli 0.6.0 (HEAD-44ff7c1, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
jormungandr 0.6.0 (HEAD-44ff7c1, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
```

**Comment by CraniumDad** [comment]
Same issue experienced here.   My node is now offline.

**Comment by WoodDesk** [comment]
I get a similar error:

Error in the overall configuration of the node
 |-> Error while parsing the node configuration file: p2p.trusted_peers[0]: invalid type: string "/ip4/3.123.177.192/tcp/3000", expected struct TrustedPeer at line 12 column 19
 |-> p2p.trusted_peers[0]: invalid type: string "/ip4/3.123.177.192/tcp/3000", expected struct TrustedPeer at line 12 column 19

**Comment by mzabaluev** [comment]
Duplicate of #945 
Our bad, we should have updated the docs with the release.

**Comment by hemants1** [comment]
May I know when the docs will be updated with Trusted peers with associated Public IDs?

**Comment by benbenek** [comment]
> May I know when the docs will be updated with Trusted peers with associated Public IDs?

An actual config has been shared in the following telegram channel:
https://t.me/CardanoStakePoolWorkgroup
